libs/ui-toolkit/src/components/dialog/dialog.tsx

```diff
@@ -37,8 +37,9 @@ export function Dialog({
     {
       'lg:w-[620px] w-full': size === 'small',
       'w-full w-full md:w-[720px] lg:w-[940px]': size === 'medium',
+      'w-full w-full md:w-[80vw] lg:w-[80vw]': size === 'large',
       'left-[0px] top-[99px] h-[calc(100%-99px)] border-0 translate-x-[0] translate-y-[0] border-none overflow-y-auto w-full':
-        size === 'large',
+        size === 'extra-large',
```

Comment:
I needed something in between `medium` and `large`. I am super open to suggestions.